### PR TITLE
pyanaconda: remove code paths around module command parsing as this was deprecated

### PR DESCRIPTION
**Pull request by KKoukiou**
Remove also the related tests as these are no longer relevant.

### CODE REVIEW

File: pyanaconda/modules/payloads/kickstart.py [hunk 1]
```diff
@@ -302,6 +267,7 @@ class PayloadKickstartSpecification(KickstartSpecification):
     commands_data = {
         "ModuleData": COMMANDS.ModuleData,
         "RepoData": COMMANDS.RepoData,
+
```
Comment (M4rtinK) [inline pyanaconda/modules/payloads/kickstart.py:270]: I think this might be a typo/leftover ? :)

**Review by M4rtinK — APPROVED** [review]
Looks good to me. :)

File: pyanaconda/modules/payloads/kickstart.py (hunk 1 above)
Comment (KKoukiou) [reply pyanaconda/modules/payloads/kickstart.py:270]: I think we can also remove these once it pykickstart it gets to RemovedCommand. 
I tried removing it and I got ~50 tests failures.

**Review by vojtechtrefny — APPROVED** [review]

**Comment by KKoukiou** [comment]
The builds where successful - click links to see. There was some hickups with github events, therefore status was not updated.

**Pull request merged**